app/views/mail_shared/call_to_action/_email_reply.html.haml

```diff
@@ -0,0 +1,9 @@
+- # pass in show_approval_actions bool and an approval instance
+- if show_step_actions
+  = render partial: "mail_shared/call_to_action/button", locals: { link_path: generate_approve_url(step), 
```

Comment:
I find it a bit easier to read if the `locals` arg is put on the next line and then all locals are on the same line. What do you think?

eg:

now: 

```ruby
render partial: "mail_shared/call_to_action/button", locals: { link_path: generate_approve_url(step),
                       link_text: step.action_name}
```
vs:

```ruby
render partial: "mail_shared/call_to_action/button",
  locals: { link_path: generate_approve_url(step), link_text: step.action_name}
```